File: 30daysOFcode_Go/day 02.go
package main

import (
    "bufio"
    "fmt"
    "io"
    "os"
    "strconv"
    "strings"
    "math"
)

// Complete the solve function below.
func solve(meal_cost float64, tip_percent int32, tax_percent int32) {
    tip := float64(tip_percent)/100 * meal_cost
    tax := float64(tax_percent)/100 * meal_cost
    total := tip + tax + meal_cost
    total = math.Round(total)
    fmt.Println(total)
}

func main() {
    reader := bufio.NewReaderSize(os.Stdin, 1024 * 1024)

    meal_cost, err := strconv.ParseFloat(readLine(reader), 64)
    checkError(err)

    tip_percentTemp, err := strconv.ParseInt(readLine(reader), 10, 64)
    checkError(err)
    tip_percent := int32(tip_percentTemp)

    tax_percentTemp, err := strconv.ParseInt(readLine(reader), 10, 64)
    checkError(err)
    tax_percent := int32(tax_percentTemp)

    solve(meal_cost, tip_percent, tax_percent)
}

func readLine(reader *bufio.Reader) string {
    str, _, err := reader.ReadLine()
    if err == io.EOF {
        return ""
    }

    return strings.TrimRight(string(str), "\r\n")
}

func checkError(err error) {
    if err != nil {
        panic(err)
    }
}

File: 30daysOFcode_Python/day 02.py
'''#!/bin/python3
# Day 2: Operators
import math
import os
import random
import re
import sys

# Complete the solve function below.
def solve(meal_cost, tip_percent, tax_percent):
    result = meal_cost + meal_cost*(tip_percent/100) + meal_cost*(tax_percent/100) + 0.5
    print(f"{int(result)}")

meal_cost = float(input())
tip_percent = int(input())
tax_percent = int(input())

solve(meal_cost, tip_percent, tax_percent)
'''

(lambda meal_cost, tip_percent, tax_percent: print(int(meal_cost + meal_cost*(tip_percent/100) + meal_cost*(tax_percent/100) + 0.5)))(float(input()), int(input()), int(input()))
